static-site/components/nav/nav-logo.tsx

```diff
@@ -27,11 +27,15 @@ const Logo = (): React.ReactElement => (
 );
 
 const LogoType = (): React.ReactElement => {
-  const rainbowTitle = siteTitle
-    .split("")
-    .map((letter, i) => (
-      <span key={i} style={{ color: `var(--titleColor${i})` }}>{letter}</span>
-    ));
+  const rainbowTitle = siteTitle.split("").map((letter, i) => (
```

Comment:
Please don't combine the previously split lines:

```suggestion
  const rainbowTitle = siteTitle
    .split("")
    .map((letter, i) => (
```